libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -265,9 +265,6 @@ export const cryptocurrenciesById: Record<CryptoCurrencyId, CryptoCurrency> = {
     ],
     ethereumLikeInfo: {
       chainId: 43114,
-      node: { type: "ledger", explorerId: "avax" },
```

Comment:
Just to be sure about the behaviour: we don't want to keep them as a default value if CoinConfig is not set or loaded?

Reply:
There's always default values set here:
https://github.com/LedgerHQ/ledger-live/blob/feat/live-1147-evm-coin-config-explorer/libs/ledger-live-common/src/families/evm/config.ts